pkg/webhook/v1beta1/pod/inject_webhook_test.go

```diff
@@ -25,7 +25,10 @@ import (
 	"testing"
 	"time"
 
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/onsi/gomega"
+	"google.golang.org/protobuf/testing/protocmp"
```

Comment:
I am wondering, do we really need to use  protocmp? 
It looks like for some unit test we just compare 2 objects: https://github.com/kubeflow/katib/blob/master/pkg/util/v1beta1/katibconfig/config_test.go#L173
cc @tenzen-y @Electronic-Waste

Reply:
@andreyvelich Yeah, I find we can also pass the unit tests without using `protocmp`.

I use `protocmp` because of the former test cases I wrote in: https://github.com/kubeflow/katib/blob/0b4e7c178094a4eb84c2ef0cb3be1c6fdf2e3928/pkg/metricscollector/v1beta1/file-metricscollector/file-metricscollector_test.go#L324-L325

Now I have deleted it in `inject_webhook_test.go`.